### PR TITLE
Key Vault: Certificate review

### LINKED ISSUE #7779: Rename list methods
Per the arch board conversation, we should find a more descriptive name for these.

### LINKED ISSUE #7796: Changes from UX studies and API reviews
@heaths commented on [Thu Oct 10 2019](https://github.com/Azure/azure-sdk-for-net/issues/8035)

We have a separate table of proposed/required changes we're keeping cross-language internally.

### PR DESCRIPTION
**Pull request by iscai-msft**
fixes #7779, closes #7796

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by iscai-msft** [comment]
fixes #7779

File: sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_client.py [hunk 1]
```diff
@@ -210,35 +212,26 @@ def test_crud_operations(self, vault_client, **kwargs):
         client = vault_client.certificates
         cert_name = self.get_resource_name("cert")
         lifetime_actions = [
-            LifetimeAction(trigger=Trigger(days_before_expiry=90), action=Action(action_type=ActionType.auto_renew))
+            LifetimeAction(trigger=Trigger(lifetime_percentage=80), action=Action(action_type=ActionType.auto_renew))
         ]
         cert_policy = CertificatePolicyGenerated(
-            key_properties=KeyProperties(exportable=True, key_type="RSA", key_size=2048, reuse_key=True),
+            key_properties=KeyProperties(exportable=True, key_type="RSA", key_size=2048, reuse_key=False),
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_client.py]: That's actually what the service default for reuse_key, CLI had the default as True.

File: sdk/keyvault/azure-keyvault-certificates/README.md [hunk 2]
```diff
@@ -124,61 +124,55 @@ new versions of existing certificates, update certificate metadata, and delete c
...
-* [Retrieve a Certificate](#retrieve-a-certificate)
-* [Update an existing Certificate](#update-an-existing-certificate)
-* [Delete a Certificate](#delete-a-certificate)
-* [List Certificates](#list-certificates)
-* [Asynchronously create a Certificate](#asynchronously-create-a-certificate)
-* [Asynchronously list certificates](#asynchronously-list-certificates)
-
-### Create a Certificate
-`begin_create_certificate` creates a Certificate to be stored in the Azure Key Vault. If a certificate with
+* [Create a KeyVaultCertificate](#create-a-keyvaultcertificate)
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/README.md]: Yeah I see that point, I'll go switch it back, thanks!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py [hunk 3]
```diff
@@ -737,20 +735,20 @@ def merge_certificate(
...
     ):
-        # type: (...) -> Certificate
+        # type: (...) -> KeyVaultCertificate
         """Merges a certificate or a certificate chain with a key pair existing on the server.
 
         Performs the merging of a certificate or certificate chain with a key pair currently
         available in the service. This operation requires the certificates/create permission.
-        Make sure when creating the certificate to merge using create_certificate that you set
+        Make sure when creating the certificate to merge using begin_create_certificate that you set
         it's issuer to 'Unknown'. This way Key Vault knows that the certificate will not be signed
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py]: :'( switched, thanks

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py [hunk 4]
```diff
@@ -521,6 +513,8 @@ def list_deleted_certificates(self, include_pending=None, **kwargs):
                 :dedent: 8
         """
         max_page_size = kwargs.pop("max_page_size", None)
+        include_pending = kwargs.pop("include_pending", None)
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py]: fixed, thanks!

File: sdk/keyvault/azure-keyvault-certificates/README.md [hunk 5]
```diff
@@ -124,61 +124,55 @@ new versions of existing certificates, update certificate metadata, and delete c
...
+* [Retrieve a KeyVaultCertificate](#retrieve-a-keyvaultcertificate)
+* [Update Properties of an existing KeyVaultCertificate](#update-properties-of-an-existing-keyvaultcertificate)
+* [Delete a KeyvaultCertificate](#delete-a-keyvaultcertificate)
+* [List Properites of KeyVaultCertificates](#list-properties-of-keyvaultcertificates)
+* [Asynchronously create a KeyVaultCertificate](#asynchronously-create-a-keyvaultcertificate)
+* [Asynchronously list properties of KeyVaultCertificates](#asynchronously-list-properties-of-keyvaultcertificates)
+
+### Create a KeyVaultCertificate
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/README.md]: makes sense, switched in readme, thanks!

File: sdk/keyvault/azure-keyvault-certificates/samples/backup_restore_operations.py [hunk 6]
```diff
@@ -39,18 +39,18 @@
 credential = DefaultAzureCredential()
 client = CertificateClient(vault_endpoint=VAULT_ENDPOINT, credential=credential)
 try:
-    print("\n.. Create Certificate")
+    print("\n.. Create KeyVaultCertificate")
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/samples/backup_restore_operations.py]: Switched in samples too, thanks!

**Pull request merged**